hpx/hpx_start.hpp

```diff
@@ -770,6 +770,9 @@ namespace hpx
     inline bool start(std::nullptr_t f,
         int argc, char** argv, std::vector<std::string> const& cfg,
         hpx::runtime_mode mode = hpx::runtime_mode_default);
+
+    inline bool start(std::nullptr_t f, std::vector<std::string> const& cfg,
```

Comment:
Could you add the doxygen comments for this (similarly to the other functions, please?

Reply:
Sorry, didn't see this in time. Will do.